veggieseasons/lib/main.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/services.dart' show DeviceOrientation, SystemChrome;
+import 'package:veggieseasons/screens/home.dart';
+
+void main() {
+  SystemChrome.setPreferredOrientations([
+    DeviceOrientation.portraitUp,
+    DeviceOrientation.portraitDown,
+  ]);
+  runApp(new MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoApp(
+      color: Color(0xffd0d0d0),
```

Comment:
this is due to my personal preference of disliking random hex numbers, but consider defining a variable that at least describes what color this is. Or add an in-line comment to that effect

Reply:
Nice catch. Fixed.